packages/squidex/schema/schemas/research-tags.json

```diff
@@ -61,29 +61,41 @@
         "properties": {
           "fieldType": "Tags",
           "allowedValues": [
-            "Activity Assay",
+            "Preprint",
```

Comment:
made a mistake previously with the allowed values, the `Types` field should take the list of research-output types